.github/workflows/build.yml

```
@@ -0,0 +1,150 @@
+
+      - name: Run Tests and Generate Coverage Report
+        env:
+          DJANGO_SETTINGS_MODULE: project.settings.test
+        run: |
+          uv run pytest -n auto
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)